centos_7_py2py3/cloudify-wagon-builder.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+echo "Starting..."
+
+CONSTRAINTS_FILE=/packaging/constraints.txt
+REQUIRMENTS_FILE=/packaging/dev-requirements.txt
+
+if test -f /packaging/extra-packaging-instructions.sh
```

Comment:
Cleaner bash would write this as:
if [[ -f /packaging/extra-packaging-instructions.sh ]]; then
  source ...
fi